Update documentation (#1)
langleyfoxall/php-rsa-signer: README.md

$message->baz = 'boff';
```

> Next, instantiate the `Signer` class with the path to your private key. Its type will

Reviewer: ```suggestion
Next, instantiate the `Signer` class with the content of your private key. Its type will
```
